app/helpers/application_helper.rb

```diff
@@ -143,6 +143,22 @@ def origin_display_name(practice)
     end
   end
 
+  def origin_display_department_name(practice)
+      if practice.initiating_facility_type?
+        if practice.initiating_facility?
+          if practice.initiating_facility_type = 'department'
+            dept_id = practice.initiating_department_office_id
+            dept = origin_data_json['departments'][dept_id - 1]
+            dept['name']
+          end
+        else
+        ''
+        end
+      else
+        ''
+      end
+  end
+
   def origin_display(practice)
```

Comment:
Since this is strictly for departments, I think we can just do
```
def origin_display_department_name(practice)
    dept_id = practice.initiating_department_office_id
    dept = origin_data_json['departments'][dept_id - 1]
    dept['name']
end
```

Reply:
Yes - since we're confirming dept exists prior to calling this with the other change above... this works fine - change made.  Thx Josh